backend/slack_commands/interactive/form_handler.py

```
@@ -0,0 +1,295 @@
+            teams=teams,
+            selected_team=team_uid,
+            metadata=meta,
+        )
+        client.views_update(view_id=body["view"]["id"], view=view)
+
+    def _on_workflow_select(self, ack, body, client):
+        ack()
+
+    # ── View submission ──────────────────────────────────────────
+
+    def _on_submit(self, ack, body, client):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_on_submit`

Add return type annotation: `None`

(ANN202)